<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigWindow</class>
 <widget class="QWidget" name="ConfigWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>594</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea_advanced">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="contents_advanced">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>677</width>
        <height>752</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QCheckBox" name="checkBox_background">
         <property name="text">
          <string>Enable background</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_layout">
         <property name="text">
          <string>Vertical layout</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_popup">
         <property name="text">
          <string>Enable popup on mouse click</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_leftStretch">
         <property name="text">
          <string>Add stretch to left/top of the layout</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_rightStretch">
         <property name="text">
          <string>Add stretch to right/bottom of the layout</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="layout_timeFormat">
         <item>
          <widget class="QLabel" name="label_timeFormat">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Custom time format</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_timeFormat">
           <property name="toolTip">
            <string>$dddd - long weekday
$ddd - short weekday
$dd - day
$d - day w\o zero
$MMMM - long month
$MMM - short month
$MM - month
$M - month w\o zero
$yyyy - year
$yy - short year
$hh - hours (24 only)
$h - hours w\o zero (24 only)
$mm - minutes
$m - minutes w\o zero
$ss - seconds
$s - seconds w\o zero</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="layout_uptimeFormat">
         <item>
          <widget class="QLabel" name="label_uptimeFormat">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Custom uptime format</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_uptimeFormat">
           <property name="toolTip">
            <string>$dd - uptime days
$d - uptime days without zero
$hh - uptime hours
$h - uptime hours without zero
$mm - uptime minutes
$m - uptime minutes without zero</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="layout_tempUnits">
         <item>
          <widget class="QLabel" name="label_tempUnits">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Temperature units</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="spacer_tempUnits">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_tempUnits">
           <property name="minimumSize">
            <size>
             <width>180</width>
             <height>0</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string notr="true">Celsius</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">Fahrenheit</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">Kelvin</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="layout_tempDevice">
         <item>
          <widget class="QLabel" name="label_tempDevice">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Temperature devices</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="listWidget_tempDevice">
           <property name="toolTip">
            <string>Editable</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="layout_mount">
         <item>
          <widget class="QLabel" name="label_mount">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Mount points</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="listWidget_mount">
           <property name="toolTip">
            <string>Editable
del - remove item</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="layout_hddSpeedDevice">
         <item>
          <widget class="QLabel" name="label_hddSpeedDevice">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>HDD devices (speed)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="listWidget_hddSpeedDevice">
           <property name="toolTip">
            <string>Editable
del - remove item</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="layout_hddDevice">
         <item>
          <widget class="QLabel" name="label_hddDevice">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>HDD devices (temp)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="listWidget_hddDevice">
           <property name="toolTip">
            <string>Editable
del - remove item</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="layout_netdir">
         <item>
          <widget class="QLabel" name="label_netdir">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Network directory</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_netdir">
           <property name="toolTip">
            <string>&quot;/sys/class/net&quot; by default</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="layout_netdev">
         <item>
          <widget class="QCheckBox" name="checkBox_netdev">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Disable auto select device and set specified device</string>
           </property>
           <property name="text">
            <string>Set network device</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_netdev">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="editable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="layout_batdev">
         <item>
          <widget class="QLabel" name="label_batdev">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Battery device</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_batdev">
           <property name="toolTip">
            <string>&quot;/sys/class/power_supply/BAT0/capacity&quot; by default</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="layout_acOnline">
         <item>
          <widget class="QLabel" name="label_acOnline">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>AC online tag</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_acOnline">
           <property name="toolTip">
            <string>Line, which returns when AC is online</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="layout_acOffline">
         <item>
          <widget class="QLabel" name="label_acOffline">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>AC offline tag</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_acOffline">
           <property name="toolTip">
            <string>Line, which returns when AC is offline</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="layout_acdev">
         <item>
          <widget class="QLabel" name="label_acdev">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>AC device</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_acdev">
           <property name="toolTip">
            <string>&quot;/sys/class/power_supply/AC/online&quot; by default</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="spacer_advanced">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>scrollArea_advanced</tabstop>
  <tabstop>checkBox_background</tabstop>
  <tabstop>checkBox_layout</tabstop>
  <tabstop>checkBox_popup</tabstop>
  <tabstop>checkBox_leftStretch</tabstop>
  <tabstop>checkBox_rightStretch</tabstop>
  <tabstop>lineEdit_timeFormat</tabstop>
  <tabstop>lineEdit_uptimeFormat</tabstop>
  <tabstop>comboBox_tempUnits</tabstop>
  <tabstop>listWidget_tempDevice</tabstop>
  <tabstop>listWidget_mount</tabstop>
  <tabstop>listWidget_hddSpeedDevice</tabstop>
  <tabstop>listWidget_hddDevice</tabstop>
  <tabstop>lineEdit_netdir</tabstop>
  <tabstop>checkBox_netdev</tabstop>
  <tabstop>comboBox_netdev</tabstop>
  <tabstop>lineEdit_batdev</tabstop>
  <tabstop>lineEdit_acOnline</tabstop>
  <tabstop>lineEdit_acOffline</tabstop>
  <tabstop>lineEdit_acdev</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
